docs/application/native/guides/index.md

```diff
@@ -251,7 +251,7 @@ The following list defines Tizen Native API. The list describes the API modules
   | Voice control         | Provides functions for registering commands and getting notifications when a registered command is recognized. | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__MODULE.html) (since 2.4), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__MODULE.html) (since 3.0) | [Voice Control](text-input/voice-control.md) |
   | Voice control elementary  | Provides functions to control UI components through voice commands. | [mobile](../api/mobile/latest/group__VOICE__CONTROL__ELEMENTARY__MODULE.html) (since 2.4), [wearable](../api/wearable/latest/group__VOICE__CONTROL__ELEMENTARY__MODULE.html) (since 3.0) | [Voice Control](text-input/voice-control.md) |     |
   | Voice control engine  | Provides functions to operate Voice-Control Engine.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__ENGINE__MODULE.html) (since 4.0) | [Voice Control](text-input/voice-control-engine.md) |
-  | Voice control manager | Provides functions for recording voice and giving responses for recognized voice commands to users.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) (since 5.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) (since 5.0) |  |
+  | Voice control manager | Provides functions for recording voice and giving responses for recognized voice commands to users.  | [mobile](../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) (since 5.0), [wearable](../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) (since 5.0) | [Voice Control](text-input/voice-control-manager.md) |
```

Comment:
Its a request, please verify the links, I am unable to check the links as the staging.

Reply:
Can you guide me how to verify the links and which link can I verity?
I just add a link for this new guide file(`text-input/voice-control-manager.md`) and the file path is exact.

Reply:
@stom-hwang 
https://docs1.stg.tizen.org/staging/1395/application/native/guides/

After your latest push, the staging link started working, now I  am able to check links.